cmd/server.js

```diff
@@ -113,7 +119,8 @@ app.get('/street/near/geojson', function( req, res ){
           'properties': {
             'id': o.street.id,
             'name': Array.isArray( o.street.name ) ? o.street.name[0] : o.street.name,
-            'polyline': o.street.line
+            'polyline': o.street.line,
+            'distance': ( Math.floor(( o.proj.dist || 0 ) * 1000000 ) / 1000000 )
```

Comment:
this is in order to trim the numbers after the decimal point.

it's done like this; because javascript:

```javascript
( Math.floor( 5.2398427634897238947 * 1000000 ) / 1000000 )
5.239842

parseFloat( ( 5.2398427634897238947 ).toFixed(6) )
5.239843
```

I'll add a code comment